stable/weave-scope/charts/weave-scope-agent/templates/daemonset.yaml

```diff
@@ -63,6 +63,14 @@ spec:
               mountPath: /var/run/scope/plugins
             - name: sys-kernel-debug
               mountPath: /sys/kernel/debug
+      {{- if .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml .Values.nodeSelector | nindent 8 -}}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations:
```

Comment:
Is it ok to have two `tolerations:` sections?  (there's another one on line 25)
(I know nothing about helm)

Reply:
Indeed. Nice catch @bboreham Missed that daemonset had already a tolerations section. Refactoring now...

Reply:
@bboreham fixed daemonset.yaml __tolerations__ section.
Please review it again!